### PR TITLE
Internal: Update wp-env in Core 

### PR DESCRIPTION
**Pull request by TzviRabinovitch**

### CODE REVIEW

File: .github/workflows/run-wp-env/action.yml [hunk 1]
```diff
@@ -39,16 +39,17 @@ runs:
...
+          npm i -g --no-package-lock --no-save @wordpress/env@8.6.1
     - name: Setup WordPress environment
       shell: bash
       run: |
         npx wp-env start
         # change wp-env folder owner to www-data
         sudo chown 33:33 -R ~/wp-env/
-        npx wp-env run cli "bash -c 'bash elementor-config/setup.sh'"
+        npx wp-env run cli bash -c 'bash elementor-config/setup.sh'
+        ls -R /var/www/html
```
Comment (matipojo) [inline .github/workflows/run-wp-env/action.yml:50]:
```suggestion
```

File: .github/workflows/plugins-tester.yml [hunk 2]
```diff
@@ -111,7 +111,8 @@ jobs:
         with:
           PLUGINS: './tests/plugins-tester/plugins/elementor-tests-utils'
           PORT: '7777' # In order to enable local run side by side with another default server (port 8888)
-      - run: wp-env run cli "wp option update elementor_onboarded 1"
+      - run: wp-env run cli wp option update elementor_onboarded 1
+      - run: wp-env run cli mkdir /var/www/html/wp-content/upgrade
```
Comment (matipojo) [inline .github/workflows/plugins-tester.yml:115]:
```suggestion
      - run: wp-env run cli mkdir -m 777 /var/www/html/wp-content/upgrade
```

File: .github/workflows/plugins-tester.yml [hunk 3]
```diff
@@ -111,7 +111,8 @@ jobs:
         with:
           PLUGINS: './tests/plugins-tester/plugins/elementor-tests-utils'
           PORT: '7777' # In order to enable local run side by side with another default server (port 8888)
-      - run: wp-env run cli "wp option update elementor_onboarded 1"
+      - run: wp-env run cli wp option update elementor_onboarded 1
+      - run: wp-env run cli mkdir -m 777 /var/www/html/wp-content/upgrade
```
Comment (matipojo) [inline .github/workflows/plugins-tester.yml:115]:
```suggestion
      - run: wp-env run cli mkdir -p -m 777 /var/www/html/wp-content/upgrade
```

File: tests/plugins-tester/src/PluginsTester.js [hunk 4]
```diff
@@ -44,29 +41,11 @@ export class PluginsTester {
...
 		const errors = [];
 		this.options.pluginsToTest.forEach( ( slug ) => {
-			try {
-				const filename = process.env.CI ? '../../logs.txt' : 'logs.txt';
-
-				if ( fs.existsSync( filename ) ) {
-					fs.unlinkSync( filename );
-				}
-
-				this.runWP( `npx wp-env run cli 'bash elementor-config/activate_plugin.sh ${ slug } 2>>logs.txt' ` );
```
Comment (dmaziar) [inline tests/plugins-tester/src/PluginsTester.js:54]: @matipojo why did you delete this code? We spent time to display warnings as annotations.
